pallets/thea/src/lib.rs

```diff
@@ -389,17 +536,23 @@ pub mod pallet {
 		/// * `bls_signature`: BLS Signature.
 		/// * `token_type`: Token Type.
 		/// * `payload`: Encoded Deposit Payload.
+		/// * `_relayer_index`: The index of relayer in active set.
+		/// * `_network`: Network id.
+		/// * `_signature`: The signature of the relayer on the payload.
 		#[pallet::weight(1000)]
 		pub fn approve_deposit(
 			origin: OriginFor<T>,
 			bit_map: u128,
 			bls_signature: [u8; 96],
 			token_type: TokenType,
 			payload: Vec<u8>,
-		) -> DispatchResult {
-			let _relayer = ensure_signed(origin)?;
+			_relayer_index: u8,
+			_network: Network,
+			_signature: T::Signature,
```

Comment:
Why we need these args if we are not using them?

Reply:
Yes, we are using the argument to validate the tx, check [here](https://github.com/Polkadex-Substrate/Polkadex/pull/669/files#diff-db9aab8a8240e8c00955d7f7debad560f2c408d932d7ab7bbbb94575855c5dadR471)

Reply:
Could you please add comment in dockblock of each method used those fields to not remove them at some point and understand the purpose of these fields?